src/base-emulator.js

```diff
@@ -135,10 +135,14 @@ class BaseEmulator {
    */
   _setEnviromentVariables() {
     process.env.DATASTORE_EMULATOR_HOST = `${this._options.host}:${this._options.port}`;
-
+    process.env.DATASTORE_EMULATOR_HOST_PATH = `${this._options.host}:${this._options.port}/datastore`
+    process.env.DATASTORE_HOST = `http://${this._options.host}:${this._options.port}`
+    process.env.DATASTORE_DATASET = 'test'
+    
     if (!process.env.DATASTORE_PROJECT_ID && this._options.project) {
       process.env.DATASTORE_PROJECT_ID = this._options.project
     }
+    process.env.GCLOUD_PROJECT = process.env.DATASTORE_PROJECT_ID
```

Comment:
The `GCLOUD_PROJECT` environment variable is not listed in the referenced document and  it is marked as deprecated in a few documentation. Not clear for me it is necessary so please remove it

Reply:
It seems the environment variables in the existing codebase are not set as to spec. See https://gist.github.com/mdornseif/d9bd0c55a7aab298d0a29bee124302c8 which fails on me.